mono/mini/llvm-jit.cpp

```diff
@@ -284,13 +285,14 @@ class MonoLLVMJIT {
 		initializeScalarOpts(registry);
 		initializeInstCombine(registry);
 		initializeTarget(registry);
+		linkCoreCLRGC();
 
 		const char *opts = g_getenv ("MONO_LLVM_OPT");
 		if (opts == NULL) {
 			// FIXME: find optimal mono specific order of passes
 			// see https://llvm.org/docs/Frontend/PerformanceTips.html#pass-ordering
 			// the following order is based on a stripped version of "OPT -O2"
-			opts = " -simplifycfg -sroa -lower-expect -instcombine -licm -simplifycfg -lcssa -indvars -loop-deletion -gvn -memcpyopt -sccp -bdce -instcombine -dse -simplifycfg";
+			opts = " -simplifycfg -sroa -lower-expect -instcombine -licm -simplifycfg -lcssa -indvars -loop-deletion -gvn -memcpyopt -sccp -bdce -instcombine -dse -simplifycfg -place-safepoints";
```

Comment:
Don't we need to add `-place-safepoints` unconditionally irrespective of `MONO_LLVM_OPT`?

Reply:
Makes sense! Fixed